third_party/remoteapis/BUILD.bazel

```diff
@@ -1,31 +1,30 @@
 load("@rules_java//java:defs.bzl", "java_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//third_party/grpc:build_defs.bzl", "java_grpc_library")
+load("//tools/build_rules:utilities.bzl", "java_library_srcs")
 
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
-exports_files(["LICENSE"])
-
-load("@io_bazel//third_party/grpc:build_defs.bzl", "java_grpc_library")
-load("@io_bazel//tools/build_rules:utilities.bzl", "java_library_srcs")
+exports_files(["bazel_remote_apis.patch"])
 
 filegroup(
     name = "srcs",
     srcs = glob(["**"]),
     visibility = ["@//third_party:__pkg__"],
 )
 
-JAVA_LIBRARY_PROTOS = [
-    "build_bazel_semver_semver",
-    "build_bazel_remote_asset_v1_remote_asset",
-    "build_bazel_remote_execution_v2_remote_execution",
-]
+JAVA_LIBRARY_PROTOS = {
+    "build_bazel_semver_semver": "@bazel_remote_apis//build/bazel/semver:semver_java_proto",
+    "build_bazel_remote_asset_v1_remote_asset": "@bazel_remote_apis//build/bazel/remote/asset/v1:remote_asset_java_proto",
+    "build_bazel_remote_execution_v2_remote_execution": "@bazel_remote_apis//build/bazel/remote/execution/v2:remote_execution_java_proto",
+}
 
 [java_library_srcs(
-    name = proto + "_java_proto_srcs",
-    deps = [":" + proto + "_java_proto"],
-) for proto in JAVA_LIBRARY_PROTOS]
+    name = proto[0] + "_java_proto_srcs",
+    deps = [proto[1]],
+) for proto in JAVA_LIBRARY_PROTOS.items()]
```

Comment:
It's a matter of style, but it seems like we could just inline these and the resulting code wouldn't be much longer.